public/content/contributing/translation-program/translatathon/index.md

```diff
@@ -2,7 +2,7 @@
 title: Ethereum.org Translatathon
 description: Join the first ethereum.org Translatathon to contribute to ethereum.org, learn about Ethereum, and compete for prizes.
 lang: en
-template: event
+# template: event
```

Comment:
This is addressed more completely in #80

Reply:
... in the meantime, this simple commenting-out of the template should kick it over to use Static instead